frontend/src/app/Settings/Users/ManageUsers.tsx

```diff
@@ -73,29 +81,7 @@ const ManageUsers: React.FC<Props> = ({
     updateIsUserEdited(true);
   }
 
-  const onSaveChanges = (userId: string) => {
-    onUpdateUser(usersState[userId]); // TODO this does nothing atm
```

Comment:
i don't have context on why you're removing this, but just reiterating that updateUser() is an available mutation on the backend

Reply:
Thanks for restating -- that is a good point to be explicit about. This particular code block was a placeholder. Like you noticed, I went with `addUserToCurrentOrg` for its implementation assuming the intent is for org admins to add users themselves. If that is not how we plan on using the app in the next couple weeks, then yeah, would need to replace/add in `addUser` instead

@aliciabeckett-gov thoughts on this